ban/http/resources.py

```diff
@@ -16,38 +16,116 @@ class BaseCollection:
     DEFAULT_LIMIT = 20
     MAX_LIMIT = 100
 
-    def get_limit(self, req):
-        return min(int(req.params.get('limit', self.DEFAULT_LIMIT)),
-                   self.MAX_LIMIT)
+    def get_start(self, req, count):
+        """Get start value from request.
+
+        Keyword arguments:
+            req -- Request.
+
+        Business rules:
+            if start == not defined then start = 0.
+            if start < 0            then start = 0.
+            if start > count        then start = 0.
+            else start = start.
+        """
+        try:
+            start = int(req.params.get('start', 0))
```

Comment:
You should be able to use `req.get_param_as_int`: http://falcon.readthedocs.io/en/latest/api/request_and_response.html#falcon.Request.get_param_as_int